about-me.html

```diff
@@ -0,0 +1,172 @@
+    </p>
+
+    <p>
+        <img src="/img/me_forum_php_2017.jpeg" title="Picture of myself, speaking at Forum PHP 2017" />
+    </p>
+
+    <p>
+        Software quality, scalability and Service Oriented Architecture are subjects
+        that fascinate me. I do my best to improve myself in these fields but I also
+        try to open my perspective to other domains like compilation and static analysis.
+    </p>
+
+    <p>
+        Generally speaking, I love working and sharing on topics related to software
+        architecture and software quality. I like to see myself as a passionate developer,
+        an Open Source enthusiast and an eternal learner.
+    </p>
+
+    <h2>What I am looking for</h2>
+
+    <p>
+        <b>Challenge motivates me.</b> Even if I have mainly worked on PHP and
+        Ruby applications, I crave for opportunities to discover new
+        technologies and concepts.
+    </p>
+
+    <p>
+        <b>I love learning.</b> If you use programming languages or tools that I
+        have not already mastered, I probably already heard about it and I definitely
+        am confident that I can quickly become proficient with it.
+    </p>
+
+    <p>
+        <b>I am interested in various topics.</b> Pure development, architecture and
+        clean code, DevOps practices, scalability and performance concerns, monitoring
+        and SRE are all topics that I am interested in. And there are a lot more!
+    </p>
+
+    <h2>I know things</h2>
+
+    <p>
+        <b>I am a good developer.</b> I am really efficient with PHP and I know
+        its ecosystem pretty well. Regexes don't frighten me and neither do tricky
+        SQL requests or an obscure server configuration.
+        <a href="https://github.com/K-Phoen/">My code</a> is clean, tested and
+        meant to be readable by humans. In addition to <a href="https://github.com/K-Phoen?tab=repositories&language=php">PHP</a>,
+        I know my way around
+        <a href="https://github.com/K-Phoen?tab=repositories&language=ruby">Ruby</a>
+        and Bash and I have played with
+        <a href="https://github.com/K-Phoen?tab=repositories&language=python">Python</a>,
+        <a href="https://github.com/K-Phoen?tab=repositories&language=go">Go</a>
+        and I am interested by Rust.
+    </p>
+
+    <p>
+        <b>I am curious.</b> When I find something interesting or curious, I
+        take the time to investigate it. That's how I found out <a href="https://blog.kevingomez.fr/til/2015/07/26/why-is-uniqid-slow/">why PHP's
+        <code>uniqid()</code> function is "<em>slow</em>"</a>.
+    </p>
+
+    <p>
+        <b>I am not afraid of specs/RFCs.</b> My knowledge of the
+        <a href="https://www.editeur.org/83/Overview/">Onix standard</a>
+        and my contributions to TEA's e-book metadata server should be enough
+        to prove it. From time to time, I also refer to RFCs like the
+        <a href="https://www.editeur.org/83/Overview/">RFC 2616</a>,
```

Comment:
I think the link is incorrect here, or you really like ONIX ;)

Reply:
I should probably mention that I have PTSD because of the Onix standard…

(nice catch btw, thanks!)